src/lib.rs

```diff
@@ -1966,14 +1975,15 @@ impl Build {
                     cmd.push_opt_unless_duplicate(format!("-O{}", opt_level).into());
                 }
 
-                if cmd.is_like_clang() && target.contains("windows") {
+                if cmd.is_like_clang() && target.os == "windows" {
                     // Disambiguate mingw and msvc on Windows. Problem is that
                     // depending on the origin clang can default to a mismatchig
                     // run-time.
-                    cmd.push_cc_arg(format!("--target={}", target).into());
+                    // FIXME: Convert rustc target to Clang target.
+                    cmd.push_cc_arg(format!("--target={raw_target}").into());
```

Comment:
this won't work on cc's msrv 1.63

Reply:
Yes it will? This has been possible since 1.58: https://blog.rust-lang.org/2022/01/13/Rust-1.58.0.html#captured-identifiers-in-format-strings

Reply:
Oh sorry my memory is wrong 😅

Reply:
Oh sorry my memory is wrong 😅